orchprovider/k8s/v1/util_test.go

```diff
@@ -171,3 +171,37 @@ func TestNS(t *testing.T) {
 		}
 	}
 }
+
+// TestIsInCluster tests the output of IsIncluster func
+func TestIsInCluster(t *testing.T) {
+
+	cases := []struct {
+		name           string
+		expectedOutput bool
+		expectedError  error
+	}{
+		{"default", true, nil},
```

Comment:
Is there a negative test to add?

Reply:
i dont think we can cover the negative side because the functions that are getting called always returns nil

Reply:
the nil is hard coded there